### PR TITLE
feat(pools): add purebtc.com

### PR DESCRIPTION
**Pull request by intech**

### CODE REVIEW

**Comment by dubuqingfeng** [comment]
ok

File: pools.json [hunk 1]
```diff
@@ -535,6 +535,10 @@
         "/ArkPool/": {
             "name": "ArkPool",
             "link": "https://www.arkpool.com/"
+        },
```
Comment (ryanendo) [inline pools.json:538]: 0x4eC6bc9691F2742C86307dA42f1F0b280a7c5D0F

**Review by ryanendo — APPROVED** [review]
Nice

File: pools.json [hunk 2]
```diff
@@ -535,6 +535,10 @@
...
+        "/PureBTC.COM/" : {
+            "name": "PureBTC.COM",
+            "link": "https://purebtc.com"
```
Comment (ryanendo) [inline pools.json:541]: 0x4eC6bc9691F2742C86307dA42f1F0b280a7c5D0F

**Review by ryanendo — APPROVED** [review]

File: pools.json (hunk 2 above)
Comment (luozhanxin) [reply pools.json:541]: ?

Comment (intech) [reply pools.json:541]: > ?

this is some kind of spam

**Pull request closed without merging**